[2024-06-21T15:29:46.659+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: kafka_spark_dag.kafka_data_stream scheduled__2024-06-21T15:24:45.634546+00:00 [queued]>
[2024-06-21T15:29:46.665+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: kafka_spark_dag.kafka_data_stream scheduled__2024-06-21T15:24:45.634546+00:00 [queued]>
[2024-06-21T15:29:46.666+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2024-06-21T15:29:46.674+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): kafka_data_stream> on 2024-06-21 15:24:45.634546+00:00
[2024-06-21T15:29:46.680+0000] {standard_task_runner.py:57} INFO - Started process 20527 to run task
[2024-06-21T15:29:46.683+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'kafka_spark_dag', 'kafka_data_stream', 'scheduled__2024-06-21T15:24:45.634546+00:00', '--job-id', '269', '--raw', '--subdir', 'DAGS_FOLDER/dag_kafka_spark.py', '--cfg-path', '/tmp/tmpoictw0on']
[2024-06-21T15:29:46.684+0000] {standard_task_runner.py:85} INFO - Job 269: Subtask kafka_data_stream
[2024-06-21T15:29:46.706+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 269 for task kafka_data_stream (DagRun for kafka_spark_dag with run_id or execution_date of 'scheduled__2024-06-21T15:24:45.634546+00:00' not found; 20527)
[2024-06-21T15:29:46.714+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
